docs/install.rst

```diff
@@ -42,7 +46,24 @@ need to install Python 3 with virtualenv in your system as well.
     Users of other Linux distributions may need to install the equivalent
     packages, depending on their system configuration.
 
-.. _Homebrew: http://brew.sh/
+.. note::
+
+   If you want full support for searching inside your Read the Docs
+   site you will need to install Elasticsearch_.
+
+   Ubuntu users could install this package as following::
+
+        sudo apt-get install elasticsearch
+
+.. note::
+
+   Besides the Python specific dependencies, you will also need
+   PostgreSQL_ and Redis_.
+
+   Ubuntu users could install this package as following::
+
+        sudo apt-get install redis-server postgresql
+
```

Comment:
You don't need Postgres actually, we don't have any Postgres-specific code that I know of. The development database is fine, but not recommended for production instances. 

We do require Redis currently, though only because of a feature that prints our build queue in our templates. I think we could get rid of this however -- I'll open a ticket to discuss this. I think it's fine to leave the docs on redis in for now.